demos/plane/plane.js

```diff
@@ -200,12 +200,20 @@ Plane.rows1st = 0;
 Plane.rows2nd = 0;
 
 /**
- * Redraw the rows when the slider has moved.
+ * Redraw the rows and update blocks when the slider has moved.
  * @param {number} value New slider position.
  */
 Plane.sliderChange = function(value) {
   var newRows = Math.round(value * 410 / 20);
   Plane.redraw(newRows);
+
+  function updateBlocks(blocks, verbose) {
```

Comment:
is verbose getting used for anything?

Reply:
whoops, it was leftover from debugging. Will remove.